.github/lock-unlock-issues.yml

```diff
@@ -0,0 +1,11 @@
+'🚦 status: awaiting triage':
+  issues:
+      comment: >
+          To reduce notifications, issues are locked until they are https://github.com/EddieHubCommunity/LinkFree/labels/%F0%9F%8F%81%20status%3A%20ready%20for%20dev and to be assigned. You can learn more in our contributing guide https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md
+      lock : true
+
+'🏁 status: ready for dev':
+  issues:
+      comment: >
+            The thread has been temporarily unlocked.Please follow our community guidelines.
```

Comment:
You are free to change the comment,GitHub actions display while unlocking the issue.